### DISCUSSION TITLE
Export DataGrid to Excel

### DISCUSSION THREAD

**Discussion by GdiTocco** [Megabit/Blazorise#1671] [Ideas]
Is there the possibility to export a DataGrid to an Excel file?
I think, at least for me, it would be very useful

**Comment by stsrki** [comment]
There is no possibility to export data to an Excel file. To add that feature it would mean to add a third-party dependency which I don't wish to do with Blazorise. Not to mention that it would be difficult to support every type of data structure that users could have. So no, there is no plan to add support for any exporter.

**Comment by d-oit** [comment]
Check out:
https://github.com/radzenhq/radzen-blazor/blob/master/RadzenBlazorDemos/Controllers/ExportController.cs
